### PR TITLE
Updated updateUser fields to match the new update form in mygfw

### PR DESCRIPTION
**Pull request by blayhem**
## Overview

New fields matching the ones in "Update profile" in mygfw. Main updates:
- Name in two fields now
- Sector options (job title, company...)
- "Geographic area of interest"
- Newsletter topics & interests ("What topics are you interested in?")

### CODE REVIEW

**Comment by ikas** [comment]
> I can't see any major issues. Maybe worth checking with @tiagojsag or @ikas

LGTM as well, good job :)

File: app/src/services/googleSheetsService.js [hunk 1]
```diff
@@ -41,11 +41,11 @@ class GoogleSheetsService {
                     'agreed_to_test': 'yes',
                     'Date First Added': this.getDate(),
                     'Email': user.email,
-                    'First': user.fullName,
+                    'First': user.firsName,
```
Comment (tiagojsag) [inline app/src/services/googleSheetsService.js]: typo

**Comment by tiagojsag** [comment]
rebase please

**Comment by aabdaab** [comment]
added tests ☝️ @blayhem could you take a look at the test cases, let me know if I didn't cover something

**Comment by blayhem** [comment]
> added tests ☝️ @blayhem could you take a look at the test cases, let me know if I didn't cover something

It looks good, thanks for taking care of this! I'd say there are some endpoints missing, no? For example, `updateUser`. But maybe @tiagojsag has more knowledge on the coverage we need for this case, considering that there were no tests originally.

**Pull request merged**